polaris-react/src/components/Button/Button.scss

```diff
@@ -911,7 +911,7 @@
     }
 
     #{$se23} &:hover,
-    #{$se23} &:focus:not(:active, .pressed, .disabled) {
+    #{$se23} &:focus:not(:active):not(.pressed):not(.disabled) {
```

Comment:
Tophat of [AppIndexResourceListItem](https://admin.web.fix-not-selector-lists.aaron-casanova.us.spin.dev/store/shop6/settings/apps?tab=installed) looks good 🥇 

<img width="1131" alt="Screenshot 2023-06-29 at 6 57 18 PM" src="https://github.com/Shopify/polaris/assets/18447883/1c48c711-50f0-4ddd-8f23-e45e98a7f259">